pyproject.toml

```diff
@@ -17,7 +17,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-temporalio = "^1.7.0"
+temporalio = { path = "../sdk-python" }
```

Comment:
Cannot be committed now.  Needs to await the released Python SDK having Workflow Init support.